audiomentations/augmentations/transforms.py

```diff
@@ -1151,7 +1152,193 @@ def apply(self, samples, sample_rate):
             distorted_samples = samples
         return distorted_samples
 
+    
+class LowPassFilter(BaseWaveformTransform):
+    """
+    Apply low-pass filtering to the input audio.
+    """
+
+    supports_multichannel = True
+    requires_sample_rate = True
+
+    def __init__(
+        self,
+        min_cutoff_freq=150,
+        max_cutoff_freq=7500,
+        p: float = 0.5,
+    ):
+        """
+        :param min_cutoff_freq: Minimum cutoff frequency in hertz
+        :param max_cutoff_freq: Maximum cutoff frequency in hertz
+        :param p: The probability of applying this transform
+        """
+        super().__init__(p)
+
+        self.min_cutoff_freq = min_cutoff_freq
+        self.max_cutoff_freq = max_cutoff_freq
+        if self.min_cutoff_freq > self.max_cutoff_freq:
+            raise ValueError("min_cutoff_freq must not be greater than max_cutoff_freq")
+
+    def randomize_parameters(
+        self, samples: np.array, sample_rate: int = None
+    ):
+        """
+        :params samples: (num_channels, num_samples)
+        """
+        num_samples = samples.shape[-1]
```

Comment:
This function needs a call like `super().randomize_parameters(samples, sample_rate)` in the beginning to make sure that `self.parameters["should_apply"]` is set correctly. Otherwise `apply` won't be run :/

Reply:
Fixed.